changelogs/fragments/ansible-test-locale.yml

```diff
@@ -0,0 +1,16 @@
+major_changes:
```

Comment:
Is double-listing the CL entry under two different keys what's expected now? Seems weird...

Reply:
I suppose it is when you want it to appear in two places?

Reply:
I thought it was a bit odd as well, but decided to match the changelog entry in https://github.com/ansible/ansible/pull/78175 that @sivel created.